include/alpaka/kernel/TaskKernelCpuOmp2Blocks.hpp

```diff
@@ -1,4 +1,5 @@
-/* Copyright 2019 Benjamin Worpitz, Bert Wesarg, René Widera
+/* Copyright 2019-2020 Benjamin Worpitz, Bert Wesarg, René Widera,
+ *                     Sergei Bastrakov
```

Comment:
Followed the old mentality of 80 characters per line, should have continued on the previous line